src/components/Reuseable-Components/Presentational/Modal/Modal.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
```

Comment:
This looks good, but if we externalize the React components into a separate repo, and then import them like we normally would into an app, we can then simplify the dir structure to something like `src/components/React/Modal/Modal.js`